Assets/Scripts/Player/PlayerController.cs

```diff
@@ -646,20 +646,20 @@ public class PlayerController : Entity {
 		anim.SetTrigger("Dash");
 	}
 
-	public void StartDashAnimation(bool backwards) {
+	public void StartDashAnimation(bool backwards, bool recycle = false) {
 		preDashSpeed = Mathf.Abs(rb2d.velocity.x);
 
 		// back dash animation always comes from the initial dash, where the speed boost has already been applied
-		float newSpeed = (backwards ? preDashSpeed : preDashSpeed+dashSpeed);
```

Comment:
I kept the spacing here tight to not interfere with my mental parsing of that ternary conditional operator.

Reply:
Visual studio doesn't like your thinking lol.